gateway/1.9/modules/ROOT/pages/flex-review-prerequisites.adoc

```diff
@@ -7,24 +7,47 @@ endif::[]
 
 Before you download and install Anypoint Flex Gateway, review the following requirements and limits.
 
-== Permission Requirements
+== Flex Gateway Requirements
 
-Running Flex Gateway requires the following permissions:
+=== Permission Requirements
 
-* _Manage Servers_ and _Read Servers_ permissions in Runtime Manager
+To use Flex Gateway you must have these permissions:
+
+[%header,cols="20a,30a,50a"]
+|===
+| Permission | Ability | Notes
+| Manage Servers | Create, update, and delete server and Flex Gateway resources.| Runtime Manager permission which is assigned to a specific environment in a Business Group.
```

Comment:
```suggestion
| Manage Servers | Create, update, and delete server and Flex Gateway resources.| Runtime Manager permission, which is assigned to a specific environment in a Business Group.
```

Reply:
(globally) Did you try to specify the permission? For example, did you mean?:
Runtime Manager permission that is assigned to a specific environment in a Business Group.